app/javascript/components/FilterSlider.js

```diff
@@ -0,0 +1,188 @@
+import React, { useState } from 'react';
+import { Slider, Rail, Handles, Tracks, Ticks } from 'react-compound-slider';
+
+import { Handle, Track, Tick } from './slider/components';
+
+const sliderStyle = {
+  margin: '5%',
+  position: 'relative',
+  width: '90%'
+};
+
+const railStyle = {
+  position: 'absolute',
+  width: '100%',
+  height: 14,
+  borderRadius: 7,
+  cursor: 'pointer',
+  backgroundColor: 'rgb(155,155,155)'
+};
+
+/**
+ * Component for slider to filter numerical facets, e.g. organism age
+ *
+ * TODO:
+ * - Support unit-less numerical facets, e.g. bmi, number_of_reads (SCP-2201)
+ * - Support rate units, e.g. small_molecule_perturbation__concentration (SCP-2201)
+ */
+export default function FilterSlider(props) {
+
+  const facet = props.facet
```

Comment:
I like this idea of assigning variables to properties. It makes things more readable and lets developers know what to expect.